README.md

```diff
@@ -71,6 +71,24 @@ You will be prompted to supply a username, email, and password.
 
 Having completed that process, you will be able to access the admin interface at `/admin` using your new superuser account.
 
+### Create fake finding data
```

Comment:
```suggestion
### Create sample finding data
```
:)

Reply:
To be clear: I'm suggesting this edit because I think some people may read unintended negative implications into the word `fake`.